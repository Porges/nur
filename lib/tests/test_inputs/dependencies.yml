version: "1.0"

default: 
  dependencies: [world]

world:
  dependencies: [hello]
  run: ["echo world"]

hello:
  run: ["echo hello!"]
