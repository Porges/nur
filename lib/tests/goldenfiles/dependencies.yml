version: "1.0"

default: 
  dependencies: [world]

world:
  dependencies: [hello]
  cmds: ["echo world"]

hello:
  cmds: ["echo hello!"]
